apps/ca.c

```diff
@@ -806,15 +806,20 @@ int ca_main(int argc, char **argv)
     /*
      * EVP_PKEY_get_default_digest_name() returns 2 if the digest is
      * mandatory for this algorithm.
+     *
+     * That call may give back the name "UNDEF", which has these meanings:
+     *
+     * when def_ret == 2: the user MUST NOT specify a digest
+     * when def_ret == 1: the user MAY not specify a digest
```

Comment:
I'm no native English speaker, so please take this feedback with a grain of salt: Should this comment not better use the word "MIGHT" instead of "MAY"? "...may not specify..." sounds like a mandate; "...might not specify..." more like the option (it is meant to be).

Reply:
I hear you suggest that I reword it to be like the documentation.  Good suggestion, will do